lib/src/models/transaction_chunk.dart

```diff
@@ -23,8 +23,8 @@ class TransactionChunk {
 
 @JsonSerializable()
 class TransactionOffsetResponse {
-  final String size;
-  final String offset;
+  final String? size;
```

Comment:
What's the purpose of making these optional rather than setting default magnitudes of 0?